packages/data-stores/src/wpcom-plans-ui/actions.ts

```diff
@@ -9,4 +11,19 @@ export const resetStore = () =>
 		type: 'WPCOM_PLANS_UI_RESET_STORE',
 	} as const );
 
-export type WpcomPlansUIAction = ReturnType< typeof setShowDomainUpsellDialog | typeof resetStore >;
+export const setSelectedStorageOptionForPlan = ( {
+	addOnSlug,
+	planSlug,
+}: {
+	addOnSlug: WPComStorageAddOnSlug;
+	planSlug: PlanSlug;
+} ) =>
+	( {
+		type: 'WPCOM_PLANS_UI_SET_STORAGE_ADD_ON_FOR_PLAN',
+		addOnSlug,
+		planSlug,
+	} as const );
+
+export type WpcomPlansUIAction = ReturnType<
+	typeof setShowDomainUpsellDialog | typeof resetStore | typeof setSelectedStorageOptionForPlan
```

Comment:
I think we have a bit of inconsistent naming across these state pieces now:

- action named `setSelectedStorageOptionForPlan`
- selector named `getStorageAddOnForPlan`
- reducer named `selectedStorageAddOnsForPlans`
- action const/type named `WPCOM_PLANS_UI_SET_STORAGE_ADD_ON_FOR_PLAN`

Can we pick one and be consistent, or was there a reason we haven't unified these?

@jeyip

Reply:
Thanks for pointing this out. I'd be happy to fix these 👍 

I'll create a follow-up where we can address naming conventions and any other concerns shortly

Reply:
https://github.com/Automattic/wp-calypso/pull/80874